javaScript/js-index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script  src="first.js" > </script>
+    
+</head>
+<body>
+    <h1>Enter text to check for Plindrone</h1>
+    <input id="givenText" type="text"  placeholder="None">
+    <button onclick="document.querySelector('h2').innerHTML = checkString(givenText.value)">Check</button>
+   
+    <br>
+    <br>
+    <hr>
+    <br>
+    <h1>Enter number to check for Prime</h1>
+    <input id="givenNumber" type="text"  placeholder="None">
+    <button onclick="document.querySelector('h2').innerHTML = primeCheck(givenNumber.value)">Check</button>
+    <br>
+    <br><br>
+    <hr>
+    <br>
+    <h2></h2>
```

Comment:
Don't use h2 here. I think to display anything we can use either paragraph or span